modules/gitops-release-notes-1-13-0.adoc

```diff
@@ -0,0 +1,119 @@
+
+* Before this update, users were unable to connect to remote Git repositories through a SOCKS5 proxy server because it was only compatible with HTTPS URLs. With this update, users can access remote Git repositories by using a SOCKS5 proxy server that supports the SSH protocol. link:https://issues.redhat.com/browse/GITOPS-3710[GITOPS-3710]
+
+* Before this update, the `forceHttpBasicAuth` repository configuration was not initialized properly during repository updates from Argo CD Web UI. As a result, the value was reset to default, leading to connection errors, particularly for Azure DevOps repositories. This update fixes the issue by adding a change in the upstream to ensure that the `forceHttpBasicAuth` configuration is correctly initialized during repository update operations. link:https://issues.redhat.com/browse/GITOPS-3706[GITOPS-3706]
+
+* Before this update, the Argo CD CLI would designate the initial source as the main source for multi-source applications. With this update, the Argo CD CLI supports multi-source applications by considering all sources during processing. link:https://issues.redhat.com/browse/GITOPS-2623[GITOPS-2623]
+
+* Before this update, the RolloutManager controller did not create ServiceMonitor. With this update, the RolloutManager controller is improved to include support for ServiceMonitor creation. As a result, you can now gather metrics from Rollouts using the enhanced RolloutManager controller, and these metrics are visible in monitoring. link:https://issues.redhat.com/browse/GITOPS-3271[GITOPS-3271]
+
+[id="deprecated-features-1-13-0_{context}"]
+== Deprecated and removed features
+
+* In {gitops-title} 1.13, {gitops-title} Application Manager command-line interface (CLI), `kam`, is deprecated and is planned to be removed in the future release. Red{nbsp}Hat will provide bug fixes and support for this feature during the current release lifecycle, but this feature will no longer receive enhancements and will be removed. As an alternative to {gitops-title} Application Manager CLI, `kam`, you can use Argo CD CLI that is available from {gitops-title} Operator v1.12. link:https://issues.redhat.com/browse/GITOPS-4466[GITOPS-4466]
+
+* Breaking change: Before this update, Argo Rollouts instances in {gitops-title} were installed as namespace-scoped. This means that when you created a `RolloutManager` CR within a namespace, that Argo Rollouts instance would only have permissions to deploy within that specific namespace. This issue has been fixed in this update. Now, Argo Rollouts instances in {gitops-title} are installed as cluster-scoped, which means that a RolloutManager CR created within a namespace can be deployed across the entire cluster. link:https://issues.redhat.com/browse/RHDEVDOCS-6022[RHDEVDOCS-6022]
```

Comment:
```suggestion
* Breaking change: Before this update, Argo Rollouts instances in {gitops-title} were installed as namespace-scoped. Previously, when you created a `RolloutManager` CR within a namespace, that Argo Rollouts instance would only have permissions to deploy within that specific namespace. This issue has been fixed in this update. Now, because Argo Rollouts instances in {gitops-title} are installed as cluster-scoped, a RolloutManager CR created within a namespace can be deployed across the entire cluster. link:https://issues.redhat.com/browse/RHDEVDOCS-6022[RHDEVDOCS-6022]
```

Reply:
Fixed.